tests/rados/test_track_availibility_status.py

```diff
@@ -0,0 +1,604 @@
+            return 1
+
+        random_pool = random.choice(new_pool_names)
+        msg_random_pool = f"The random pool selected is - {random_pool}"
+        log.info(msg_random_pool)
+        result_all_tests = test_all_values(client, random_pool)
+        if not result_all_tests:
+            msg_error = f" The track availability tests are failed on {random_pool}"
+            log.error(msg_error)
+            return 1
+
+        msg_pool_deletion = f" The {random_pool} is selected to delete the pool "
+        log.info(msg_pool_deletion)
+        method_should_succeed(rados_object.delete_pool, random_pool)
+        after_delete_pool_details = get_pool_availability_status(client)
+        if random_pool in after_delete_pool_details:
+            msg_error = f" The {random_pool} pool details exists in the availability-status output"
+            log.error(msg_error)
+            return 1
+        msg_info = (
+            f" The {random_pool} pool details not exists in the availability-status output."
+            f"The pool available status output is -{after_delete_pool_details}"
+        )
+        log.info(msg_info)
+        log.info("Deleting the created pools from the cluster")
+        for tmp_pool_name in new_pool_names:
+            method_should_succeed(rados_object.delete_pool, tmp_pool_name)
+        # Checking the log messages
+
+        log.info(
+            "==== Case 4: Verification of the score value displayed on multiple pools  completed===="
+        )
+        # TODO
+        log.info(
+            "=== [IN-Progress] Case 5: Verification of track availability messages in the logs==="
+        )
+
+        log.info(
+            "=== Case 6: Verification of track availability tests during recovery state==="
+        )
+
+        cmd_get_replicat_size = f"ceph osd pool get {pool_name} size"
+        output = rados_object.run_ceph_command(cmd=cmd_get_replicat_size)
+        current_replicate_size = output["size"]
+
+        cmd_increase_replica_size = (
+            f"ceph osd pool set {pool_name} size {current_replicate_size + 1} "
+        )
+        rados_object.run_ceph_command(cmd_increase_replica_size)
+        rados_object.bench_write(pool_name=pool_name, byte_size="5K", max_objs=500)
+
+        timeout = datetime.timedelta(minutes=15)
+        start_time = datetime.datetime.now()
+        recovering_found = False
+
+        while datetime.datetime.now() - start_time < timeout:
+            out_put = rados_object.run_ceph_command(cmd="ceph -s")
+            for state in out_put["pgmap"]["pgs_by_state"]:
+                if "recovering" in state["state_name"]:
+                    recovering_found = True
+                    break
+                time.sleep(2)
+            if recovering_found:
+                break
+        if not recovering_found:
+            log.error(
+                "The cluster is not in recovery state. Not executing the further tests"
+            )
+            return 1
```

Comment:
wait for clean PGs on the pool post size increase

Reply:
Included the code for waiting the PGs to clean state.